cypress/cypress/e2e/04_submariner_addon.cy.js

```diff
@@ -0,0 +1,31 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2023 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { clusterSetMethods } from '../views/clusterset/clusterset'
+
+describe('submariner - validate submariner addon tab', {
+    tags: ['@submariner'],
+    retries: {
+        runMode: 0,
+        openMode: 0,
+    }
+}, function () {
+    before(function () {
+        cy.setAPIToken()
+        cy.clearOCMCookies()
+        cy.login()
+    })
+
+    it('test the Submariner add-ons', { tags: ['addon'] }, function () {
+        let clusterSetName = (Math.random() + 1).toString(36).substring(2)
```

Comment:
Thanks for making the change.
Could you just append the random string you are generating to a "submariner" prefix.
So it will looks like - "submariner-<random_string>.

Reply:
Sure, no problem.